examples/utils.py

```diff
@@ -4,10 +4,43 @@
 import os
 import importlib.util
 import settings
+import urllib
 from IPython.display import display, JSON
 import json
 
-# IMPORTS
+
+# GENERIC UTILITIES
```

Comment:
Maybe let's have a `utils` folder at the top level, and then `generic` and `materials` inside those?

Reply:
Utils is now its own folder, updated the imports of the notebooks to match this